.github/workflows/ISSUE_TEMPLATE/bug-report.yaml

```diff
@@ -0,0 +1,18 @@
+name: Bug Report
+description: Bug with the frontend? Report it here!
+title: "[Bug]: "
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report! Before smashing the submit button please review the template.
+        Also, please also ensure that this is not a duplicate issue :)
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: Also tell us what you expected to happen and any ways to reproduce the behavior.
+      placeholder: A clear and concise description of what the 🐞 is.
+    validations:
+      required: true
```

Comment:
This is really good! Since this is a browser-based front-end app, what might also be super helpful is:

- Osmosis Frontier or Osmosis Main App?
- Add a selector for iOS, Android, desktop, Keplr mobile
- Add a selector for which browser it happened in (keep an N/A option)

Github has [a good example of this](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms#about-yaml-syntax-for-issue-forms). Version ID is irrelevant obviously if everyone is running the same thing, but getting these few more pieces might help triage.